bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -468,8 +475,9 @@ fi
 if [ "$isnlo" -gt "0" ]; then
   if grep -q -e "\$DEFAULT_PDF_SETS" -e "\$DEFAULT_PDF_MEMBERS" $CARDSDIR/${name}_run_card.dat; then
     echo "INFO: Using default PDF sets for 2017 production"
-          sed "s/\$DEFAULT_PDF_SETS/306000,322500,322700,322900,323100,323300,323500,323700,323900,292200,292600,305800,315000,13100,13163,13167,13000,13065,13069,13200,25200,25300,25000,42780,90200,91200,90400,91400,61100,61130,61200,61230,13400,82000/" $CARDSDIR/${name}_run_card.dat > ./Cards/run_card.dat
```

Comment:
I would actually duplicate the lines and comment the "full list", otherwise it will be painful to re-insert them

Reply:
But it's all version controlled... I can easily find them and add them back from this discussion.
 It's 322500,322700,322900,323100,323300,323500,323700,323900.